aries-site/src/pages/components/card.mdx

```diff
@@ -98,6 +109,80 @@ TBD
 - Cards that are less promotional should still have enticing titles but do not necessarily need as strong of visual emphasis. The white brand-approved textures are appropriate if a subtle visual emphasis is desired.
   - A use case is a software upgrade announcement.
```

Comment:
On the site, I see this sub-bullet point skip a line. Do we want to bring that up to be consistent with the other sub-bullet points that are directly below the parent bullet point?